README.markdown

```diff
@@ -179,57 +224,72 @@ Tells Puppet whether the key should or shouldn't exist. Valid options: 'present'
 
 For more on this parameter, see the [Purge existing values section](#purge-existing-values) under Usage.
 
+
 #### `registry_value`
 
-##### `path`
+Manages individual Registry values.
 
-*Optional.* Specifies a Registry value for Puppet to manage. Valid options: a string containing a Registry path. If any of the parent keys in the path don't exist, Puppet creates them automatically. For example: `HKLM\Software` or `HKEY_LOCAL_MACHINE\Software\Vendor`. Default value: the title of your declared resource.
+The title of the resource determines the path and value name that will be managed, for example:
 
-If Puppet is running on a 64-bit system, the 32-bit Registry key can be explicitly managed using a prefix. For example: `32:HKLM\Software\Value3`.
+The manifest below manages the value called 'foo' in the key 'HKLM\Software'
 
-##### `ensure`
+``` puppet
+registry_value { 'HKLM\Software\foo':
+  ...
+}
+```
 
-Tells Puppet whether the value should or shouldn't exist. Valid options: 'present' and 'absent'. Default value: 'present'.
+You can also manage the default value of a key by adding a backslash at the end of the path.  The manifest below manages the default value in the key 'HKLM\Software\Vendor'
 
-##### `type`
+``` puppet
+registry_value { 'HKLM\Software\Vendor\\':
+  ...
+}
+```
 
-*Optional.* Sets the data type of the specified value. Valid options: 'string', 'array', 'dword', 'qword', 'binary' and 'expand'. Default value: 'string'.
+Use a double backslash between the registry path and value name when managing a value with a back slash in the name.  For example the manifest below
+manages a value called 'A\Backslash' in the registry path 'HKLM\Software\Vendor'
 
-##### `data`
+``` puppet
+registry_value { 'HKLM\Software\Vendor\\\A\Backslash':
+  ...
+}
+```
 
-Provides the contents of the specified value. Valid options: a string by default; an array if specified through the `type` parameter.
+Note you must use '\\\' in the path because backslashes must be escaped if the next character is a backslash.
 
-#### `registry::service`
 
-##### `ensure`
+**Parameters**
 
-Tells Puppet whether the service should or shouldn't exist. Valid options: 'present' and 'absent'. Default value: 'present'.
+All parameters are required unless otherwise stated.
 
-##### `display_name`
+##### `path`
 
-*Optional.* Provides a Display Name for the service. Valid options: a string. Default value: the title of your declared resource.
+The path to the registry value to manage.  For example:
+'HKLM\Software\Value1', 'HKEY_LOCAL_MACHINE\Software\Vendor\Value2'. If Puppet is running on a 64-bit system, the 32-bit registry key can
+be explicitly manage using a prefix.  For example: '32:HKLM\Software\Value3'. Use a double backslash between the value name
+      and path when managing a value with a backslash in the name.
```

Comment:
Line break?

Reply:
Fixed. copy-pasta fail from the type documentation